lib/facter/util/file_helper.rb

```diff
@@ -15,12 +15,14 @@ def safe_read(path, default_return = '')
           default_return
         end
 
-        def safe_readlines(path, default_return = [])
-          return File.readlines(path, encoding: Encoding::UTF_8) if File.readable?(path)
+        # rubocop:disable Style/SpecialGlobalVars
+        def safe_readlines(path, default_return = [], sep = $/, chomp: false)
+          return File.readlines(path, sep, chomp: chomp, encoding: Encoding::UTF_8) if File.readable?(path)
```

Comment:
Can you add the `safe_readlines` changes to single commit (along with changes to call sites and tests)? Also we generally try to avoid mixing positional and keyword args.  Perhaps make `sep` a keyword arg? We can't change `path` or `default_return` as there are probably custom facts that call it.

Reply:
I broke this out into a single commit. I kept sep as a positional argument, since that mirrors the function readlines, which it is wrapping. However, if you would still like it to be a keyword arg, I am happy to make that change.

Reply:
ah no worries, thanks!